invenio_app_rdm/theme/assets/semantic-ui/js/invenio_app_rdm/landing_page/CommunitiesListModal/CommunitiesListModal.js

```diff
@@ -0,0 +1,63 @@
+// This file is part of InvenioRDM
+// Copyright (C) 2023 CERN.
+//
+// InvenioRDM is free software; you can redistribute it and/or modify it
+// under the terms of the MIT License; see LICENSE file for more details.
+
+import { i18next } from "@translations/invenio_app_rdm/i18next";
+import PropTypes from "prop-types";
+import React, { Component } from "react";
+import { Header, Modal, Button } from "semantic-ui-react";
+import { RecordCommunitiesSearch } from "./RecordCommunitiesSearch";
+
+export class CommunitiesListModal extends Component {
+  constructor(props) {
+    super(props);
+
+    this.state = {
+      modalOpen: false,
+    };
+  }
+
+  render() {
+    const { modalOpen } = this.state;
+    const { trigger } = this.props;
+
+    return (
+      <Modal
+        role="dialog"
+        aria-labelledby="record-communities-header"
+        id="community-selection-modal"
+        closeIcon
+        closeOnDimmerClick={false}
+        open={modalOpen}
+        onClose={() => this.setState({ modalOpen: false })}
+        onOpen={() => this.setState({ modalOpen: true })}
+        trigger={React.cloneElement(trigger, {
```

Comment:
just out of curiosity: why did you have to use the `cloneElement` for the trigger?

Reply:
You're right, it's not needed! I just copied the modal from somewhere else where this was implemented, and wasn't thinking :D